Importing on lodash template now, not whole library.

chrism opened the pull request:
Hi,

I've followed the advice here:

https://github.com/ember-cli/ember-welcome-page/issues/34

but made a small change, the template is imported directly like this in index.js

```js
  var template = require('lodash.template');
  var compiled = template(contents);
```

The tests pass and I've tested it on a brand new Ember project too.

I think it relates to this issue too

https://github.com/ember-cli/ember-welcome-page/issues/33

Thanks

stefanpenner commented on .gitignore:
```diff
@@ -15,3 +15,5 @@
 npm-debug.log
 testem.log
+
+.DS_Store
```
This should be part of you system gitignore, not part of the projects

stefanpenner commented:
one comment to be addressed, otherwise this is great.

chrism commented:
Ah sorry, because I saw existing .DS_Store files I thought it might be good practice.

I've removed it from the .gitignore and squashed it into one commit, hope thats good now.

Thanks for being patient :)
